airflow-core/src/airflow/ui/src/components/TruncatedText.tsx

```diff
@@ -31,7 +31,6 @@ export const TruncatedText = ({ text, ...rest }: Props) => (
       WebkitLineClamp: 2,
     }}
     title={text}
-    width="200px"
```

Comment:
This is to not impose a fixed with. If the text can expand, it will, if it cannot it will wrap and then ellipse:
![Screenshot 2025-03-24 at 12 03 16](https://github.com/user-attachments/assets/e4b60531-c68b-4c13-9c2e-5346ba9b1ea4)
![Screenshot 2025-03-24 at 12 03 26](https://github.com/user-attachments/assets/b91f29be-25bc-45be-b725-515c84e5755a)



![Screenshot 2025-03-24 at 12 03 37](https://github.com/user-attachments/assets/028e1d08-3206-4947-a878-ea20c09ae94c)